app/utils/BFS.js

```diff
@@ -0,0 +1,122 @@
+const pathFind = (field, startP, endP) => {
+	const path = [];
+
+	// tests if the point is inside the tested field				
+	var inField = function (point) {
```

Comment:
Don't var
A boolean or a function returning one should start with a predicate: "isInField". Predicates: is, has, did, does, can, will, should, etc.